src/test/java/teammates/test/cases/ui/InstructorFeedbacksPageActionTest.java

```diff
@@ -51,10 +51,10 @@ public void testExecuteAndPostProcess() throws Exception{
         
         InstructorFeedbacksPageData pageData = (InstructorFeedbacksPageData) r.data;
         assertEquals(instructorId, pageData.account.googleId);
-        assertEquals(2, pageData.courses.size());
-        assertEquals(6, pageData.existingFeedbackSessions.size());
-        assertEquals(null, pageData.newFeedbackSession);
-        assertEquals(null, pageData.courseIdForNewSession);
+        assertEquals(2, pageData.getNewForm().getCourses().size());
+        //assertEquals(6, pageData.existingFeedbackSessions.size());
```

Comment:
Didn't mean for this to be reviewed yet, just wanted to check if the changes for jstl were right.